src/routes/auth-token/create.js

```diff
@@ -16,15 +16,7 @@ export default {
 
   parameters: Joi.object().keys({
 
-    // NOTE: the length of the signedData may change if the ethereumjs-util
-    //  package is ever updated and the fromRpcSignature method changes it's
-    //  length validation, as indicated by a comment in the source
-    //
-    // see: https://github.com/ethereumjs/ethereumjs-util/blob/fde15dd/index.js
-    //
-    // NOTE: signedData should not have the 0x prefix as Joi will validate it as
-    //  a raw Buffer
-    signedData: Joi.binary().encoding('hex').length(65).required(),
+    signedData: Joi.string().length(132).required(),
```

Comment:
Will length always be 132? Is there any documentation/source code you can point me to that verifies this will always be 132? Could always just remove the length too.

Reply:
Will let `recoverTypedSignature` deal with length if it needs to.